grub.cfg

```diff
@@ -0,0 +1,7 @@
+set debug=linux
+serial --speed=115200 --word=8 --parity=no --stop=1
+terminal_input --append  serial
+terminal_output --append serial
+
+linux (ahci0,msdos1)/isar-image-xen-debian-buster-qemuamd64.vmlinuz-4.19.36 root=/dev/nfs rw ip=dhcp nfsroot=192.168.4.190:/srv/nfs/debian,vers=3,udp nfsrootdebug --- console=ttyS0,115200 earlyprint=serial,ttyS0,115200
```

Comment:
kernel on a hard drive with nfsroot @pietrushnic ?

Reply:
@miczyg1 yes I placed kernel on HDD, can I somehow pull that from http server? My GRUB2-fu is limited.

Reply:
https://www.gnu.org/software/grub/manual/grub/grub.html#Network
When there is a need, there is also a solution. See also section 13.1.

Reply:
@miczyg1 did you tested that for me or just googled? Under grub I can't see any network interfaces, do you know that issue?